src/components/course/tests/CourseHeader.test.jsx

```diff
@@ -108,6 +110,40 @@ describe('<CourseHeader />', () => {
     expect(screen.queryByText(shortDescription)).toBeInTheDocument();
   });
 
+  test.each([
+    [['test-skill1', 'test-skill2', 'test-skill3', 'test-skill4'], false],
+    [['test-skill1', 'test-skill2', 'test-skill3', 'test-skill4', 'test-skill5', 'test-skill6'], true],
+  ])('renders course skills', (skills, skillsGreaterThanLimit) => {
+    const courseState = {
+      ...initialCourseState,
+      course: {
+        ...initialCourseState.course,
+        skillNames: skills,
+      },
+    };
+
+    render(
+      <CourseHeaderWithContext
+        initialAppState={initialAppState}
+        initialCourseState={courseState}
+        initialUserSubsidyState={initialUserSubsidyState}
+      />,
+    );
+    skills.forEach((skill, index) => {
+      if (skillsGreaterThanLimit && index >= initialCourseState.maxVisibleSkills) {
+        expect(screen.queryByText(skill)).not.toBeVisible();
+      } else {
+        expect(screen.queryByText(skill)).toBeInTheDocument();
+        expect(screen.queryByText(skill)).toBeVisible();
+      }
+    });
+    if (skillsGreaterThanLimit) {
+      expect(screen.queryByText('show more')).toBeInTheDocument();
```

Comment:
this text should ideally be a constant exported from the component that contains the ui code for the course-skills (ideally a new component, but at least from CourseHeader)

Reply:
Fixed.